projects/legacy/components/input-time/input-time.component.ts

```diff
@@ -110,6 +110,12 @@ export class TuiInputTimeComponent
         this.nativeFocusableElement.value = value;
     }
 
+    public checkOption(option: TuiTime): void {
+        if (option.toAbsoluteMilliseconds() === this.value?.toAbsoluteMilliseconds()) {
```

Comment:
I wonder if we can use valueOf to make it shorter

Reply:
fixed